test/parallel/test-repl-.editor.js

```diff
@@ -31,6 +31,17 @@ function run({input, output, event}) {
   stream.emit('data', input);
   replServer.write('', event);
   replServer.close();
+
+  if (!checkTerminalCodes) {
+    while (found.includes(terminalCode))
+      found = found.replace(terminalCode, '');
```

Comment:
@addaleax quoted terminal code is sufficient right?
`const terminalCodeRegex = /\u001b\[1G\u001b\[0J> \u001b\[3G/g`